ansible/roles/telegraf/tasks/main.yml

```diff
@@ -14,5 +14,10 @@
 - name: Add telegraf to PATH
   win_path:
     elements:
-    - 'C:\Program Files\telegraf'
+    - '{{ telegraf_path }}'
     state: present
+
```

Comment:
What do you think about moving the starting of telegraf to deploy task, so the template wouldn't send statistics to graphite?